src/elements/OCAvatar.vue

```diff
@@ -10,6 +22,13 @@ export default {
   status: "review",
   release: "1.0.0",
   props: {
+    /**
+     * Defines if an avatar has been loaded
+     */
+    loaded: {
+      type: Boolean,
+      required: false,
```

Comment:
add default false

Reply:
and loading might sounds more reasonable

Reply:
Right - fixed.